packages/gatsby-cli/src/create-cli.ts

```diff
@@ -534,6 +508,95 @@ Creating a plugin:
         report.log(`Telemetry collection ${enabled ? `enabled` : `disabled`}`)
       }),
     })
+    .command({
+      command: `plugin [cmd]`,
+      describe: `Useful commands relating to Gatsby plugins`,
+      builder: yargs =>
+        yargs.positional(`cmd`, {
+          choices: [`docs`],
+          describe: "Valid commands include `docs`.",
+          type: `string`,
+        }),
+      handler: handlerP(({ cmd }) => {
+        if (cmd === `docs`) {
+          console.log(`
+      Using a plugin:
+      - What is a Plugin? (https://www.gatsbyjs.com/docs/what-is-a-plugin/)
+      - Using a Plugin in Your Site (https://www.gatsbyjs.com/docs/using-a-plugin-in-your-site/)
+      - What You Don't Need Plugins For (https://www.gatsbyjs.com/docs/what-you-dont-need-plugins-for/)
+      - Loading Plugins from Your Local Plugins Folder (https://www.gatsbyjs.com/docs/loading-plugins-from-your-local-plugins-folder/)
+      - Plugin Library (https://www.gatsbyjs.com/plugins/)
+
+      Creating a plugin:
+      - Naming a Plugin (https://www.gatsbyjs.com/docs/naming-a-plugin/)
+      - Files Gatsby Looks for in a Plugin (https://www.gatsbyjs.com/docs/files-gatsby-looks-for-in-a-plugin/)
+      - Creating a Generic Plugin (https://www.gatsbyjs.com/docs/creating-a-generic-plugin/)
+      - Creating a Local Plugin (https://www.gatsbyjs.com/docs/creating-a-local-plugin/)
+      - Creating a Source Plugin (https://www.gatsbyjs.com/docs/creating-a-source-plugin/)
+      - Creating a Transformer Plugin (https://www.gatsbyjs.com/docs/creating-a-transformer-plugin/)
+      - Submit to Plugin Library (https://www.gatsbyjs.com/contributing/submit-to-plugin-library/)
+      - Source Plugin Tutorial (https://www.gatsbyjs.com/tutorial/source-plugin-tutorial/)
+      - Maintaining a Plugin (https://www.gatsbyjs.com/docs/maintaining-a-plugin/)
+      - Join Discord #plugin-authoring channel to ask questions! (https://gatsby.dev/discord/)
+                 `)
+        } else {
+          console.log(`Current valid subcommands are: 'docs'`) // right now this is hardcoded because we only have a single command
+        }
+      }),
+    })
+    .command({
+      command: `options [cmd] [key] [value]`,
+      describe: `View or set your gatsby-cli configuration settings.`,
+      builder: yargs =>
+        yargs
+          .positional(`cmd`, {
+            choices: [`set`],
+            type: `string`,
+            describe: `Configure your package manager.`,
+          })
+          .positional(`key`, {
+            choices: [`pm`],
+            type: `string`,
+            describe: `Set the package manager \`gatsby new\` is using.`,
+          })
+          .positional(`value`, {
+            choices: [`npm`, `yarn`],
+            type: `string`,
+            describe: `Set package manager as \`npm\` or \`yarn\`.`,
+          }),
```

Comment:
This will be problematic if we ever add more `key` choices that don't relate to package managers.

Not sure if we can describe that `npm`/`yarn` choices are only for cases where `pm` key is used (for the future)

Reply:
Also to clarify - this is not something I would block this PR on - that's more feedback / food for thought - use your judgement if this is valuable and warrant more work on this one

Reply:
You can get optional args depending on other args so it will become a mess but we can make it work